Create apt sofacy.txt

jangeisbauer opened the pull request.

meni-braun commented on Campaigns/apt sofacy.txt:
```diff
@@ -0,0 +1,7 @@
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\*.dat",*'
```
I'm not sure if this regex is logically correct, since `*` has a different meaning in RegEx - so I think it won't capture anything.
Can you try it against real data, see if the query catches anything?

jangeisbauer replied:
looks like its working in general ...:

![2019-06-19 11_36_00-Clipboard](https://user-images.githubusercontent.com/6720053/59754588-ac917e80-9286-11e9-8bfd-e698f309d24e.png)

meni-braun replied:
But the meaning of this regex is:
find a string that contains "conhost.exe" followed by zero or more "e" characters.

So I say the current query has is a logical bug.
@'rundll32.exe %APPDATA%\*.dat",*'

This one translates to:
Look for the string "rundll32.exe %APPDATA%\" followed by zero or more `"` characters, followed by any character (that's the dot) followed by `dat"," followed by zero or more `,` characters.

jangeisbauer replied:
you are right. I changed * to .*

meni-braun commented on Campaigns/apt sofacy.txt:
```diff
@@ -0,0 +1,7 @@
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\.*\.dat\",.*'
```
the `.*` at the end is redundant, because `matches regex` operator captures partial text

meni-braun replied:
Since `.` means "any character" in regex, you should change each `.` to `\.` meaning in regex real dot. Applies to the `.exe` in `rundll32.exe`

meni-braun commented on Campaigns/apt sofacy.txt:
```diff
@@ -0,0 +1,7 @@
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\.*\.dat\",.*'
```
Regarding `%APPDATA%\.*`
it could be that regex engine will interpret that as `\.` = which means a real dot. To work around this wrap the dot with `[ ]` like this: `%APPDATA%\[.]*\`

jangeisbauer commented:
Ok, I'll figure that out :-). I made some tests:

Folderpath search for --> C:\Windows\System32\conhost.exe

where FolderPath matches regex @'conhos.*' -->match
where FolderPath matches regex @'conhos\.*' -->match
where FolderPath matches regex @'.*Windows.*conhost.exe' -->match
where FolderPath matches regex @'Windows.*conhost.exe' -->match
where FolderPath matches regex @'Wind.*System32.*' -->match
where FolderPath matches regex @'Wind.*System32' -->match

- So as you suggested: I removed the .* at the end
- I replaced . with \. in rundll32.exe
- I decided to just remove the \ from %APPDATA%
